src/utils/web3-utils.js

```diff
@@ -1,14 +1,27 @@
 import { ethers, providers as Providers } from 'ethers'
 import { getPreferredChain } from '../local-settings'
+import {
+  ARBISCAN_NETWORK_TYPES,
+  ARBISCAN_TYPES,
+  BLOCKSCOUT_NETWORK_TYPES,
+  BLOCKSCOUT_TYPES,
+  ETHERSCAN_NETWORK_TYPES,
+  ETHERSCAN_TYPES,
+  POLYGONSCAN_NETWORK_TYPES,
+  POLYGONSCAN_TYPES,
+} from './provider-types'
 
 const DEFAULT_LOCAL_CHAIN = ''
```

Comment:
Should we add a default local chain :thinking: ?

Reply:
not sure, we don't have any local environment, what do you think?

Reply:
Let's leave it like this for now